presto-docs/src/main/sphinx/release/release-0.272.rst

```diff
@@ -0,0 +1,55 @@
+=============
+Release 0.272
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix parameter ordering for prepared statement queries using a WITH clause.
+* Improve performance by enabling :doc:`/optimizer/cost-based-optimizations` by default.
+* Add :func:`secure_random()` function to return a cryptographically secure random number.
+* Add ability to stream data for partial aggregation instead of building hash tables.
+    It improves the performance of aggregation when the data is already ordered by the group-by keys.
+    This can be enabled with the ``streaming_for_partial_aggregation_enabled`` session property or the ``streaming-for-partial-aggregation-enabled`` configuration property.
+* Add an adaptive stage scheduling policy that switches to phased execution mode once a query's stage count exceeds a configurable upper bound.
+    This can be enabled by setting the session property ``execution_policy`` to ``phased`` and the stage count limit can be configured by the session property ``max_stage_count_for_eager_scheduling``.
+
+Hive Changes
+____________
+* Fix Parquet data page offset calculation in Parquet writer.
```

Comment:
It would be good to figure out what this means.  Perhaps "Fix error in Parquet writer".